src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -54,11 +63,14 @@ def __init__(self, #pylint: disable=too-many-arguments
     def __getattribute__(self, name):
         val = object.__getattribute__(self, name)
         if val is None:
-            raise CloudEndpointNotSetException("The endpoint '{}' for this cloud is not set but is used".format(name)) #pylint: disable=line-too-long
+            raise CloudEndpointNotSetException("The endpoint '{}' for this cloud "
```

Comment:
Since you added it in other places, should there be a `\` here for the line continuation?